examples/attention/opencog.conf

```diff
@@ -0,0 +1,137 @@
+#
+# This file provides an example configuration for a production OpenCog
+# server. Particularly noteworthy is the list of automatically loaded
+# modules.
+#
+SERVER_PORT           = 17001
+LOG_FILE              = /tmp/cogserver.log
+
```

Comment:
Or even in the `opencog/experiments/attention.` directory would be fine, (or better) as you wrote in the original pull request.  Or even `experiments/attention`, without the leading `opencog` in front, would be better still, perhaps.